src/azure-cli/HISTORY.rst

```diff
@@ -9,6 +9,7 @@ Release History
 
 2.0.20
 ++++++
+* Allow creating Linux function apps
```

Comment:
can you back out this change? Rather you should update [app service's history](https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-appservice/HISTORY.rst)

Reply:
oh, sorry I just searched for the first history.rst, didn't know there were module specific ones. Fixed.